File: Algorithm/2/selectionSort.cpp
#include <iostream>
using namespace std;
void selectionSort(int arr[], int n){
  for(int i = 0; i < n; i++)
    in minIndex = i;
    for(int j = i+1; j < n; j++)
      if(arr[j] < arr[minIndex])
        minIndex = j;
    swap(arr[i], arr[minIndex]);
}

File: Algorithm/2/selectionSort.py
def selectionSort(alist):
    n = len(alist)
    for i in range(n):
        minIndex = alist[i]
        for j in range(i+1, n):
            if alist[j] < alist[minIndex]:
                minIndex = j
        alist[i], alist[minIndex] = alist[minIndex], alist[i]
    return alist
l = [1, 3, 4, 2, 5, 6, 1]
l = selectionSort(l)
print(l)
